compiler/rustc_borrowck/src/diagnostics/conflict_errors.rs

```diff
@@ -666,15 +666,17 @@ impl<'cx, 'tcx> MirBorrowckCtxt<'cx, 'tcx> {
         let tcx = self.infcx.tcx;
 
         // Find out if the predicates show that the type is a Fn or FnMut
-        let find_fn_kind_from_did =
-            |predicates: ty::EarlyBinder<&[(ty::Predicate<'tcx>, Span)]>, substs| {
-                predicates.0.iter().find_map(|(pred, _)| {
+        let find_fn_kind_from_did = |predicates: ty::EarlyBinder<
+            &[(ty::Predicate<'tcx>, Span)],
+        >,
+                                     substs| {
```

Comment:
Go home rustfmt, you're drunk

Reply:
Hehehe, we could add a FIXME: re run fmt when bug #XYZ is fixed.

Reply:
This appears to be caused be the let_chain inside the closure, as removing it causes it to be formatted correctly for me.